src/main/java/step1/Tokenizer.java

```diff
@@ -0,0 +1,21 @@
+package step1;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Tokenizer {
+    private final int splitValueIndex = 1;
+    private final int stringIndex = 2;
+    private final static String defaultSplitValue = ",|;";
+    private final static Pattern pattern = Pattern.compile("//(.)\n(.*)");
```

Comment:
해당 피드백을 참고해주시면 감사하겠습니다 :)
https://github.com/next-step/java-lotto/pull/911#discussion_r525843063

```suggestion
    private static final int SPLIT_VALUE_INDEX = 1;
    private static final int STRING_INDEX= 2;
    private static final String DEFAULT_SPLIT_VALUE = ",|;";
    private static final static Pattern PATTERN= Pattern.compile("//(.)\n(.*)");
```